src/mame/konami/simpsons.cpp

```diff
@@ -593,16 +598,15 @@ INPUT_PORTS_END
 
 void simpsons_state::object_dma()
 {
-	int counter, num_inactive;
-	uint16_t *src, *dst;
+	uint16_t *dst;
 
 	m_k053246->k053247_get_ram(&dst);
 
-	src = m_spriteram.get();
-	num_inactive = counter = 256;
+	uint16_t *src = m_spriteram.get();
+	int num_inactive = 256, counter = 256;
```

Comment:
Can it be `uint16_t const *src;`?  I don’t think it should be writing to the source, just updating the pointer.